standardization/src/main/resources/application.conf.template

```diff
@@ -17,3 +17,8 @@ standardized.hdfs.path="/tmp/conformance-output/standardized-{0}-{1}-{2}-{3}"
 
 #system-wide time zone
 timezone="UTC"
+#time zone used for timestamp fields standardization if none specified in the schema
+#defaultTimestampTimeZone
```

Comment:
Does this equal nothing?

Reply:
Default is UTC, set by the code in TimeZoneNormalizer, right?

Reply:
> 
> 
> Does this equal nothing?

Sorry. Added a better example.

Reply:
> 
> 
> Default is UTC, set by the code in TimeZoneNormalizer, right?

Default is the value of `timezone` setting. Which in the current state of code should not deviate from _UTC_.